Updated folder path (#1382)
Unity-Technologies/com.unity.multiplayer.docs: mppm/virtual-players/virtual-players-enable.md

To use a Virtual Player to test multiplayer gameplay, activate at least one Virtual Player.

> The first time you enable a Virtual Player Unity creates a new folder inside your project folder  `...[ProjectName]/Library/VP` that contains the data for each Virtual Player you activate. When you deactivate a Virtual Player, Unity caches its corresponding folder to use again when you re-activate it.

Reviewer: ```suggestion
The first time you enable a Virtual Player, Unity creates a new folder within your project folder at `...[ProjectName]/Library/VP`. This folder stores the data for each Virtual Player you activate. When you deactivate a Virtual Player, Unity caches its corresponding folder to use again when you re-activate it.
```
